rb/BUILD.bazel

```diff
@@ -139,15 +165,11 @@ ruby_library(
         ":manager-macos",
         ":manager-windows",
         ":license",
-    ],
-    deps = [
```

Comment:
Do we really do away with all these dependencies?

Reply:
Those were actually not necessary in the first place.